### PR TITLE
Workshop survey ui

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  {text: 'Overall Success Score (out of 6)', key: 'overall_success'},
+];
+
+const SurveyResultsHeader = React.createClass({
+  contextTypes: {
+    router: React.PropTypes.object.isRequired
+  },
+
+  propTypes: {
+    workshops: React.PropTypes.arrayOf(
```
Comment (mehalshah) [inline apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx]: Might be worth moving workshop to a shape in workshopConstants

File: dashboard/test/ui/features/level_group_multi_page_dots.feature [hunk 2]
```diff
@@ -42,6 +42,7 @@ Scenario: Submit three pages as... 1. some, 2. none, 3. all questions answered.
 
   # Go back to the first page to see that correct options are selected.
   Then I am on "http://learn.code.org/s/allthethings/stage/23/puzzle/2/page/1?noautoplay=true"
+  And I wait for 2 seconds
   And element ".level-group-content:nth(0) #checked_2" is visible
   And element ".level-group-content:nth(1) #checked_1" is visible
   And element ".level-group-content:nth(2) #checked_2" is visible
```
Comment (aoby) [inline dashboard/test/ui/features/level_group_multi_page_dots.feature:49]: unrelated?

File: apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+          name: React.PropTypes.string.isRequired,
+          email: React.PropTypes.string.isRequired
+        }).isRequired
+      })
+    )
+  },
+
+  getInitialState() {
+    return {
+      course: '',
```
Comment (aoby) [inline apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx]: I think this was my fault from an earlier conversation, but this should match the key in setState. I'd suggest changing this to `selectedCourse` but I'd also accept changing the one on line 65 to `state`.

Comment (aoby) [reply apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx]: Also `selectedWorkshopId` should be here. It's not necessary, but generally good form to supply an initial value to all keys that appear in `setState`

File: apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+import $ from 'jquery';
+import _ from 'lodash';
+import React from 'react';
+import {
+  Row,
+  Col,
+  Button
+} from 'react-bootstrap';
+import moment from 'moment';
+import 'react-select/dist/react-select.css';
```
Comment (aoby) [inline apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx]: Is this needed? It doesn't look like you're importing the associated 'react-select` control that uses this css.

**Comment by mehalshah** [comment]
image

File: dashboard/test/ui/features/level_group_multi_page_dots.feature (hunk 2 above)
Comment (mehalshah) [reply dashboard/test/ui/features/level_group_multi_page_dots.feature:49]: Fixed

File: apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx (hunk 4 above)
Comment (mehalshah) [reply apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx]: Removed

**Comment by aoby** [comment]
LGTM, pending suggested changes

File: apps/src/code-studio/pd/workshop_dashboard/types.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  course: React.PropTypes.string.isRequired,
+  enrolled_teacher_count: React.PropTypes.number.isRequired,
+  capacity: React.PropTypes.number.isRequired,
+  facilitators: React.PropTypes.array.isRequired,
+  organizer: React.PropTypes.shape({
+    name: React.PropTypes.string.isRequired,
+    email: React.PropTypes.string.isRequired
+  }).isRequired
+});
+
```
Comment (aoby) [inline apps/src/code-studio/pd/workshop_dashboard/types.js:17]: nice :)
